### PR TITLE
Add Vue example

### PR DESCRIPTION
**Pull request by johman10**
I found your help wanted tag on issue here: https://github.com/mmckeaveney/theme-it/issues/7

I figured to put myself on the Vue example and here is the PR related to it.
If you have any feedback on how I'm doing stuff in the example let me know.

I hope you like it. Let me know if you have any feedback. :)

I marked myself as a code contributor, but maybe this is more documentation?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -294,6 +296,13 @@ themeit.createThemes({
 themeit.applyTheme("nested-light-theme");
 ```
 
+## Examples
+As a way of show casing the power of themeit we created some examples in different frameworks.
+The examples are just as a showcase. Themeit doesn't, in any way, depend or the frameworks below.
+
+### Vue
+A basic Vue example of Theme It can be found here: https://codesandbox.io/s/zz2on366jl?view=preview
```
Comment (shogunpurple) [inline README.md]: If you could just update this to be a link to your example that would be great. 

Like this - [Vue](https://codesandbox.io/s/zz2on366jl?view=preview)

File: README.md [hunk 2]
```diff
@@ -294,6 +296,13 @@ themeit.createThemes({
 themeit.applyTheme("nested-light-theme");
 ```
 
+## Examples
+As a way of show casing the power of themeit we created some examples in different frameworks.
+The examples are just as a showcase. Themeit doesn't, in any way, depend or the frameworks below.
```
Comment (shogunpurple) [inline README.md]: typo

**Review by shogunpurple — CHANGES_REQUESTED** [review]
Just a few minor comments. Great work, thank you so much for this PR! 🚀 Once the comments are addressed, I'll merge this 👍

**Comment by johman10** [comment]
@mmckeaveney I changed the things you requested. Feel free to have another look and merge it when you feel like it's ready. :)

**Review by shogunpurple — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**